libs/sdk/src/Constants.ts

```diff
@@ -10,7 +10,7 @@ export const AUTO_COLLECT_ADVERT_ID = false;
 export const TRACK_LIFECYCLE_EVENTS = true;
 export const RECORD_SCREEN_VIEWS = false;
 export const LOG_LEVEL = RUDDER_LOG_LEVEL.ERROR;
-export const SDK_VERSION = '1.9.0';
+export const SDK_VERSION = '1.10.0';
```

Comment:
If that's the package.json value you can import it and read it directly

Reply:
Thanks, will research it.

Reply:
I'm already planning to work on updating the version as part of another ticket in this sprint. Therefore, for now, I'll revert this change and make the necessary updates in a separate PR.